### ISSUE TITLE
[BUG] Timestamp fields will not recognized in usage with Cross Cluster Search

### ISSUE THREAD

**Issue by Crazyigor1987** [opensearch-project/anomaly-detection-dashboards-plugin#453]
**What is the bug?**
Anomaly detection cannot determine timestamp fields in usage of cross cluster search. Error message is "Timestamp field: (@timestamp) is not found in index mapping". See Screenshot:

![image](https://user-images.githubusercontent.com/16761737/229464172-5718b8e7-5d65-429e-b7ea-841f586b5345.png)

Even other existing timestamp field are not recognized.

![image](https://user-images.githubusercontent.com/16761737/229465431-cc97f770-dfe6-40ef-9d5b-9e04ebb564bd.png)

**How can one reproduce the bug?**

1. Create an Remote Cluster for Cross Cluster Search
2. Index Remote Cluster in Opensearch Dashboard
3. Open Anomaly Detector Plugin and create a new Detector with the Remote Cluster as Datasource
![image](https://user-images.githubusercontent.com/16761737/229467008-8ca70c7d-b658-4e72-ae4f-51d8b67fcdd5.png)
4. Add "@timestamp" or other valid datefield as timestamp field

**What is the expected behavior?**
Even with Cross Cluster Search and manually entried Datasource/Timestamp Fields should the anomaly detection find my Documents.

**What is your host/environment?**
Azure AKS Kubernetes v.1.25
Opensearch2.6
Opensearch-Dashboard2.6

**Comment by kaituo** [comment]
Crazyigor1987@, thanks for reporting the issue. We don't support remote index now. Will add the issue to our backlog.

**Comment by ohltyler** [comment]
@kaituo we do support remote indices - see https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/244

The difference is, then the configured timestamps and feature fields have to be manually input. On first glance, this looks to be some issue with the validation API, where it is unable to pull details from the remote indices. Is this a missed use case @amitgalitz ?

**Comment by kaituo** [comment]
@ohltyler yes, we supported it partially when fgac is not enabled. I guess validation does not consider that case.
